src/components/v5/common/ActionSidebar/partials/Motions/types.ts

```diff
@@ -11,3 +9,7 @@ export enum CustomStep {
 }
 
 export type Steps = MotionState | CustomStep;
+
+export type MotionsProps = Omit<UseGetColonyActionReturnType, 'action'> & {
```

Comment:
Why is this one wrong?

Reply:
I wouldn't extend a hook's interface directly, it's spaghettifying the codebase. Imo it's better to just use what's necessary for the component to work, otherwise we're coupling `useGetColonyAction`'s internal workings to the `Motions` component